docs/redirects-rewrites.md

```diff
@@ -1,30 +1,58 @@
 # Redirects and rewrites
-Version 4 of the Essential Next.js build plugin adds support for native Next.js [rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) and [redirects](https://nextjs.org/docs/api-reference/next.config.js/redirects). These are defined in your `next.config.js` file and include support for some features that are not included in Netlify redirects and rewrites.
+
+Version 4 of the Next Runtime adds support for native Next.js
+[rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) and
+[redirects](https://nextjs.org/docs/api-reference/next.config.js/redirects). These are defined in your `next.config.js`
+file and include support for some features that are not included in Netlify redirects and rewrites.
 
 ## Using Netlify redirects and rewrites on a Next.js site
-Every site on Netlify supports [redirects and rewrites](https://docs.netlify.com/routing/redirects/), which are defined in a `_redirects` file or `netlify.toml`, and sites that use this plugin are no exceptions. However there are some caveats to bear in mind when using them. The plugin generates several rewrites of its own, which are used to map paths from the site to different Netlify functions which handle SSR, preview mode and images, as well as assets in `/_next/static`. Any Netlify redirects or rewrites that you create [take precedence over these rewrites](#Redirect-and-rewrite-precedence), so you should avoid adding a root-level Netlify rewrite, because that would override the rewrites generated by and required by the plugin.
+
+Every site on Netlify supports [redirects and rewrites](https://docs.netlify.com/routing/redirects/), which are defined
+in a `_redirects` file or `netlify.toml`, and sites that use the Next Runtime are no exceptions. However there are some
+caveats to bear in mind when using them. The Next Runtime generates several rewrites of its own, which are used to map
+paths from the site to different Netlify functions which handle SSR, preview mode and images, as well as assets in
```

Comment:
I think the 'f' is supposed to be capitalized, but might want to get a quick confirmation from @KyleBlankRollins 

```suggestion
paths from the site to different Netlify Functions which handle SSR, preview mode and images, as well as assets in
```